docs/Repository.md

```diff
@@ -96,6 +96,7 @@ A basic ADSL only BCM6362 based gateway. Very useful as SIP ATA.
 | 2      | 15.32.6210         | 2015-10-21 | #A            | [HTTPS](https://github.com/kevdagoat/hack-technicolor/raw/master/firmware/telia-gant-1_15.32.6210-1081007-bank_dump.xz) - *note: this **IS NOT** an RBI firmware, it is a raw bank dump, you can't use with TFTP or regular firmware upgrade tools*   |
 | 2      | 16.1.7053          | 2016-10-28 | #A            | [HTTPS](https://github.com/kevdagoat/hack-technicolor/raw/master/firmware/telia-gant-1_16.1.7053-1941003-bank_dump.xz) - *note: this **IS NOT** an RBI firmware, it is a raw bank dump, you can't use with TFTP or regular firmware upgrade tools*   |
 | 2      | 17.1.7992          | 2018-05-15 | #C            | [HTTPS](https://github.com/kevdagoat/hack-technicolor/raw/master/firmware/telia-gant-1_17.1.7992-0001012-bank_dump.xz) - *note: this **IS NOT** an RBI firmware, it is a raw bank dump, you can't use with TFTP or regular firmware upgrade tools*   |
+| 2      | 17.1.7992          | 2018-05-15 | #C            | [HTTPS](https://github.com/kevdagoat/hack-technicolor/raw/master/firmware/telia_gant-1_17.1.7992-0001012-20180515151413.rbi.xz) [HTTP VLAN5](http://10.0.98.250/TG389ac_gant-1_17.1.7992-0001012-20180515151413_PROD.rbi) |
```

Comment:
Delete the rbi file form github, we don't host them here. Delete the bank dump too as we don't need it anymore.
Keep only the main original HTTP link from the ISP, remove the VLAN indication, use the * footnote instead as for others in this page (take the others form Telia as example).

Add the respective RBI torrent with that same HTTP link as webseed, upload the .torrent file here in this repo.

Reply:
Who is going to seed the torrent?

Reply:
Done